projects/hslayers/src/assets/locales/en.json

```diff
@@ -31,7 +31,9 @@
     "file": "File",
     "saveToDatabase": "Save to database",
     "URL": {
-      "selectAll": "Select all Layers"
+      "selectAll": "Select all Layers",
+      "submitLayerTitle": "Submit layer title",
+      "submitLayerName": "Submit layer name"
```

Comment:
Why "submit"? Is it submitted somewhere? Shouldn't "fill in" be a better wording?

Reply:
First thing that came tom my overheated mind. Ill redo these, thanks for pointing it out